storagemarket/impl/provider.go

```diff
@@ -265,6 +273,20 @@ func (p *Provider) HandleAskStream(s network.StorageAskStream) {
 	}
 }
 
+func (p *Provider) Configure(options ...StorageProviderOption) {
+	for _, option := range options {
+		option(p)
+	}
+}
+
+func (p *Provider) DealAcceptanceBuffer() abi.ChainEpoch {
+	return p.dealAcceptanceBuffer
+}
+
+func (p *Provider) UniversalRetrievalEnabled() bool {
```

Comment:
What is the value of these functions if they are not added to the StorageProvider interface type? Should they be? I'm not sure they should?

Oh, nm, you're using them for testing. Makes sense. +1